### DISCUSSION TITLE
HyperHDR white light flashing

### DISCUSSION THREAD

**Discussion by bartkaron** [awawa-dev/HyperHDR#646] [Q&A]
Hi everyone,
I am very new in Hyperion/HyperHDR. I did my setup with Hyperion and later I decided to change to HDR version. All works as expected. I have just one weird issue.

The white light flasing from time to time. Something like every 10 - 30 secs, sometimes much longer.

I wonder what can be the issue of that. I am using Raspberry Pi Zero W version 1 and the HDMI Video Captrue card 4K HD and HDCP 2.2. I have also set to captrue the lowest resolution in HyperHDR so it display correct colors at the right time.

This is the link to uploaded video of one flash at 3rd sec of the video:
https://www.veed.io/view/3f1ceadc-39ff-47d3-abe8-4c23887df2a5

And this one is from my performance where I am a bit worry about CPU
![Screenshot_20230927_214213_Chrome](https://github.com/awawa-dev/HyperHDR/assets/139003901/cc2d3bf8-cfa0-406a-95cc-9f0a0552545b)

Tell me if you need any other information which can be useful to debug my issue.

Thank you in advance for any help or direction.
Bart

**Comment by awawa-dev** [comment]
Hi
For performance tips there is a section in the wiki: https://github.com/awawa-dev/HyperHDR/wiki/Quick-start 
Rpi zero 1 comes from 2012 realm with only one core so don't expect much. Rpi zero 2 with 4 cores is much, much better choice. The video is not uploaded yet but the most important part is missing that may be related to the problem: the hardware configuration of the LEDs, what controls the LED strip, whether the cables are short and whether there is a common ground and a 3.3V to 5V level switch. If the problem is hardware related (improperly designed/built, incorrect or missing components, or simply damaged LEDs) you have no way to debug it yourself, so there are some examples of recommended configurations that should at least eliminate the basic errors.

**Reply by bartkaron** [reply]
Hi,
I did one extra step from the wiki link you provided and currently I cannot see any flashing but I will double check during night time so I will make sure it is gone.

I know the RPi zero 1 is weak but this was the only one which I have right now. I did not buy a better one yet. First I wanted to check if I can make HyperHDR work on my end. Currently I see it works so I think I might go to buy RPi 4 which should handle this much better.

For people who are going to look for answer I did following:
Under Video Capturing / USB Capture, I checked Quarter of frame mode, what lower down my CPU usage from red 100% to orange 60 - 80%. 

I will comment here one more time to confirm if the issue is resolved later today.

Thanks

**Reply by awawa-dev** [reply]
> Currently I see it works so I think I might go to buy RPi 4 which should handle this much better.

And the just announced RPi 5 should probably work even better :)

**Reply by AstaRom** [reply]
> Under Video Capturing / USB Capture, I checked Quarter of frame mode, what lower down my CPU usage from red 100% to orange 60 - 80%.

Hello. A friend of mine built his first Ambilight about three years ago;
He also used a Raspberry Pi Zero W (until he replaced it with the RPi 3).
It had noticeable delays due to weak hardware.
He was able to partially remove them by increasing the amount of allocated memory for the GPU to 192 Mb.

`sudo raspi-config`

- Select ”Advanced Options”
- Among the options, select “Performance Options”
- Then choose the “GPU Memory” option.
- Update the memory by typing in the number according to your memory requirement. Here, I have changed it to **192**, and press “Enter”.
- Select the “Finish” option.
- Then a prompt will appear on the screen asking you to reboot, select “yes” option to confirm rebooting the device.
